apim/4.x/build/files/graviteeio-apim-portal-ui.conf

```diff
@@ -1,9 +1,45 @@
 server {
 	listen  0.0.0.0:8085;
-	root    /opt/graviteeio/apim/portal-ui;
-	index index.html;
 
-	location / {
-		try_files $uri $uri/ /index.html;
-	}
+    add_header Content-Security-Policy "frame-ancestors 'self';" always;
```

Comment:
add a comment here to explain how to allow communication between console and portal (for the theme configuration)
https://docs.gravitee.io/pages/apim/3.x/installation-guide/upgrades/3.19.0/README.html#docker_portal_nginx_configuration